acceptance/tests/options/no_color.rb

```diff
@@ -0,0 +1,16 @@
+# This test is intended to ensure with --debug and --no-color, facter does not send any escape sequences
+# to colorize the output
+test_name "C99975: --debug and --no-color command-line options should print DEBUG messages without color escape sequences" do
```

Comment:
Nope I tried, over ssh the nothing that I did would get colorization to work.

When I logged into the console and facter from the command line, I was able to see the color change, but windows doesn't do rely on the use of an escape sequence being sent by the program that is running to change the color.